pkg/webhook/server/certificate/ca.go

```diff
@@ -0,0 +1,123 @@
+package certificate
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+)
+
+// Retrieve cluster CA bundle and encode to base 64
+func (m manager) clientCAFile() ([]byte, error) {
+	authenticationConfig := corev1.ConfigMap{}
+	err := m.crMgr.GetClient().Get(context.TODO(), types.NamespacedName{Namespace: "kube-system", Name: "extension-apiserver-authentication"}, &authenticationConfig)
+
+	if err != nil {
+		return []byte{}, errors.Wrap(err, "failed to retrieve cluster authentication config")
+	}
+	clientCaFile := authenticationConfig.Data["client-ca-file"]
```

Comment:
Please try at least manually whether this works on OpenShift too. There are people looking in knmstate there and we should make sure we don't have some hard blocker there (I expect bugs to be there due to absent CI, but at least this is an obvious potential gap)

Reply:
I did run webhook e2e tests at ocp-4.3 provider and it was passing, meaning that we have proper webhook setup I suppose this is ok then.